sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubConnectionStringProperties.java

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs;
+
+import com.azure.core.amqp.implementation.ConnectionStringProperties;
+import com.azure.core.credential.TokenCredential;
+
+import java.net.URI;
+
+/**
+ * A utility class that parses a connection string into sections. A Event Hub connection string is a set of key value
```

Comment:
Update documentation to align with product name. The product is "Event Hubs". If you are connecting to a specific instance, it would be an "Event Hub" (ie. if there is an EntityPath). 

```suggestion
 * A utility class that parses a connection string into sections. An Event Hubs connection string is a set of key-value
```

Reply:
According to your comment, fixed in new version.